nms/app/packages/magmalte/app/views/equipment/GatewayLogs.js

```diff
@@ -202,6 +216,21 @@ export default function GatewayLogs() {
   function LogsFilter() {
     return (
       <Grid container justify="flex-end" alignItems="center" spacing={1}>
+        <Grid item>
+          <FormControlLabel
```

Comment:
Move this to a separate component as mentioned earlier, so that it can be shared across components